build.py

```diff
@@ -49,9 +50,17 @@ class tclr:
 	{
 		"name" : "iOS Framework",
 		"scheme" : "ADAL",
-		"operations" : [ "build", "test", "codecov" ],
+		"operations" : [ "build" ],
 		"min_warn_codecov" : 70.0,
 		"platform" : "iOS",
+		"use_sonarcube" : "true"
```

Comment:
After some thought, it's occurred to me, making "use_sonarcube" only apply to the build step is probably a cleaner solution. There's some caching that we do on a per-target basis that will make the script go slower if we spit things out.

Reply:
ok, will make the change